projects/packages/forms/src/blocks/contact-form/components/jetpack-integrations-modal/integration-card/style.scss

```diff
@@ -26,7 +26,7 @@
 	&__title-row {
 		display: flex;
 		align-items: center;
-		gap: 8px;
+		gap: 10px;
```

Comment:
no need to increase to 10px if we remove the `margin-left` from the __connected-badge